Sources/ArgumentsUtils.swift

```diff
@@ -126,20 +134,24 @@ let bundledTemplatesPath = Path(ProcessInfo.processInfo.arguments[0]).parent() +
  - returns: The Path matching the template to find
  */
 func findTemplate(prefix: String, templateShortName: String, templateFullPath: String) throws -> Path {
-  guard templateFullPath.isEmpty else {
+  switch (templateFullPath, templateShortName) {
+  case ("", ""): throw TemplateError.noTemplateFound
```

Comment:
I'm not sure the term "Default template support" is clear enough for users not familiar with SwiftGen enough. Maybe rather say "There's not anymore a template used by default, so you can still access the bundled templates but you have to explicitly specify its name"?

Btw I'm not sure the term "bundled" that we use throughout the app (in the `swiftgen templates list` command and here) is the best choice of vocabulary either. Maybe we should use "embedded" rather than "bundled" or something similar?